content/docs/actors.md

```diff
@@ -32,6 +32,7 @@ A single isolated actor consists of source code and various settings. You can th
   * [Continuous integration]({{@link actors/development/continuous_integration.md}})
   * [Environment variables]({{@link actors/development/environment_variables.md}})
   * [Input schema]({{@link actors/development/input_schema.md}})
+  * [Secret input]({{@link actors/development/secret_input.md}})
```

Comment:
Just a question cos I'm not sure either - should we go for `secure` or `secret`?

Reply:
We have `secret` everywhere, let's do secret. If we had `secure` it would imply that the other inputs are somehow insecure, which is not the case 😄 These ones are just extra secure.